sale_financial_risk/tests/test_partner_sale_risk.py

```diff
@@ -19,10 +20,26 @@ def setUpClass(cls):
         cls.product_pricelist = cls.env["product.pricelist"].create(
             {"name": "pricelist for sale_financial_risk test"}
         )
-        cls.sale_order = cls.env["sale.order"].create(
+        cls.main_currency = cls.env.company.currency_id
+        cls.EUR = cls.env.ref("base.EUR")
+        cls.other_company = cls.env["res.company"].create(
+            {"name": "Company 2", "currency_id": cls.EUR.id}
+        )
+        cls.sale_order = cls.create_sale_order()
+        cls.env.user.lang = "en_US"
+
+    @classmethod
+    def create_sale_order(cls, currency=None, company=None):
```

Comment:
I would be explicit here, making currency and company positional arguments, so it would be more predictable and obvious.